memeolist/app/src/main/java/org/aerogear/android/app/memeolist/model/UserProfile.java

```diff
@@ -33,4 +46,12 @@ public String getPictureUrl() {
         return pictureUrl;
     }
 
+    public static UserProfile from(MemeAddedSubscription.Owner user) {
+        return new UserProfile(user.id(), user.displayname(), user.email(), user.pictureurl());
```

Comment:
We should add a `null check` on `user`

Reply:
Well, It will never be null, but I don't care to add those checks.

Reply:
Maybe mark it as @NonNull then?